draft-ietf-mls-protocol.md

```diff
@@ -3127,48 +3110,78 @@ struct {
     opaque group_id<0..255>;
     uint64 epoch;
     opaque tree_hash<0..255>;
-    opaque interim_transcript_hash<0..255>;
+    opaque confirmed_transcript_hash<0..255>;
     Extension group_context_extensions<0..2^32-1>;
     Extension other_extensions<0..2^32-1>;
-    HPKEPublicKey external_pub;
+    MAC confirmation_tag;
+    optional<HPKEPublicKey> external_pub;
```

Comment:
I would probably make this an extension (in `other_extensions`, like `ratchet_tree`) instead of an optional field.